src/pages/index.js

```diff
@@ -84,24 +90,148 @@ function handleImageClick(data) {
 function handleProfileEditSubmit(formValues) {
   userInfo.setUserInfo({
     name: formValues.title,
-    about: formValues.about,
+    about: formValues.description,
   });
+  editProfileModal.setLoading(true);
+  api
+    .setUserInfo(formValues.title, formValues.description)
+    .then((res) => {
+      userInfo.getUserInfo(res.name, res.about);
+      editProfileModal.setLoading(false);
```

Comment:
`editProfileModal.setLoading(false);`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.